src/main/java/com/iexec/worker/compute/app/AppComputeService.java

```diff
@@ -85,7 +90,7 @@ public AppComputeResponse runCompute(TaskDescription taskDescription,
                 )
                 .shouldDisplayLogs(taskDescription.isDeveloperLoggerEnabled())
                 .build();
-        // Enclave should be able to connect to the LAS
+        // Enclave should be able to connect to the LAS/the SPS
```

Comment:
```suggestion
        // Enclave should be able to connect to the LAS
```
We always need to attach a network to reach a LAS which is only accessible through the local worker network. In production SPS and CAS are accessible over Internet.